providers/web_browser.rb

```diff
@@ -267,6 +260,17 @@ def plugin_manager(username, exdir, profiledirs, plugin)
               source "web_browser_scope.js.erb"
               action :nothing
             end.run_action(:create)
+            
+            # Getting Firefox version
```

Comment:
Code for getting firefox version - that will not change during runtime - is inside a loop. This code should run only once.